pkgs/tools/networking/jwhois/default.nix

```diff
@@ -38,6 +38,8 @@ stdenv.mkDerivation (finalAttrs: {
   # TODO: this should probably be fixed at a lower level than this?
   env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.hostPlatform.isDarwin "-Wno-undef-prefix";
```

Comment:
Is this still necessary?

Reply:
Yes, I tried removing it but it's load bearing 🫠 
This is the full error, but I can't make sense of it:
```
jwhois> make[3]: Entering directory '/private/tmp/nix-build-jwhois-4.0.drv-0/jwhois-4.0/gl'
jwhois> clang -DHAVE_CONFIG_H -I. -I. -I..     -g -O2 -c dummy.c
jwhois> clang -DHAVE_CONFIG_H -I. -I. -I..     -g -O2 -c asnprintf.c
jwhois> clang -DHAVE_CONFIG_H -I. -I. -I..     -g -O2 -c printf-args.c
jwhois> clang -DHAVE_CONFIG_H -I. -I. -I..     -g -O2 -c printf-parse.c
jwhois> clang -DHAVE_CONFIG_H -I. -I. -I..     -g -O2 -c vasnprintf.c
jwhois> In file included from vasnprintf.c:38:
jwhois> In file included from ./stdio.h:30:
jwhois> ///nix/store/4s8z8il6zyq77ixy4b8kfzwsnz90vsrm-apple-sdk-11.3/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/stdio.h:220:5: error: 'TARGET_OS_IPHONE' is not defined, evaluates to 0 [-Werror,-Wundef-prefix=TARGET_OS_]
jwhois> #if TARGET_OS_IPHONE
```

Reply:
Yeah… I looked into this a while ago and couldn’t quite figure out why that stuff isn’t reliably being defined. I forget my exact conclusions though. There’s a bunch of this throughout the tree so it’s not blocking here, just would be nice to get it properly fixed at some point.